src/sql/parts/dashboard/pages/dashboardPageContribution.ts

```diff
@@ -97,33 +97,43 @@ export function generateDashboardGridLayoutSchema(type?: 'database' | 'server',
 		type: 'object',
 		properties: {
 			name: {
-				type: 'string'
+				type: 'string',
+				description: localize('tabName', "The title of the container")
 			},
-			icon: {
-				type: 'string'
+			row: {
+				type: 'number',
+				description: localize('rowNumber', "The row of the component in the grid")
 			},
-			provider: {
+			rowspan: {
+				type: 'string',
+				description: localize('rowSpan', "The rowspan of the component in the grid. Default value is 1. Use '*' to set to number of rows in teh grid.")
```

Comment:
for localization resource IDs, please prefix with the component name, like `dashboardpage.rowspan`.  These IDs need to be unique across the codebase and terms like `rowSpan` can easily collide and cause errors during XLF export process.